Lib9c/Model/Quest/QuestList.cs

```diff
@@ -82,7 +82,7 @@ EquipmentItemSubRecipeSheet equipmentItemSubRecipeSheet
         )
         {
             _quests = new List<Quest>();
-            foreach (var questData in questSheet.OrderedList)
+            foreach (var questData in questSheet.OrderedList ?? new List<QuestSheet.Row>())
```

Comment:
This changes can occur a non-deterministic error.🥲

Reply:
> This changes can occur a non-deterministic error.smiling_face_with_tear

Oh, I see. But in that case, is there are general way to handle empty input is incoming?
This is because an error occurs if we use `new QuestSheet()` to remove all the quests.